be/src/formats/parquet/file_reader.cpp

```diff
@@ -438,6 +447,80 @@ StatusOr<bool> FileReader::_filter_group(const tparquet::RowGroup& row_group) {
             }
         }
     }
+    return false;
+}
+
+bool FileReader::_filter_group_with_more_filter(const tparquet::RowGroup& row_group) {
+    // runtime_in_filter, the sql-original in_filter and is_null/not_null filter will be in
+    // _scanner_ctx->conjunct_ctxs_by_slot
+    for (auto kv : _scanner_ctx->conjunct_ctxs_by_slot) {
+        StatisticsHelper::StatSupportedFilter filter_type;
+        for (auto ctx : kv.second) {
+            if (StatisticsHelper::can_be_used_for_statistics_filter(ctx, filter_type)) {
+                const TupleDescriptor& tuple_desc = *(_scanner_ctx->tuple_desc);
+                const std::vector<SlotDescriptor*>& slots = tuple_desc.slots();
+                SlotDescriptor* slot = nullptr;
+                for (auto s : slots) {
```

Comment:
Add one interface get_slot to TupleDescriptor?

Reply:
will do it.